fh-sync-server-DEVELOPMENT.yaml

```diff
@@ -3,6 +3,17 @@ kind: Template
 metadata:
   creationTimestamp: null
   name: fh-sync-server
+  annotations:
+    openshift.io/display-name: FeedHenry Sync (Persistent)
+    description: FeedHenry Sync Server
+    iconClass: icon-nodejs
+    tags: instant-app,sync
+    template.openshift.io/long-description: Data Synchronisation by FeedHenry.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
```

Comment:
as this is upstream, does it make sense to have Red Hat? I would think FeedHenry would be OK here.